plugins/generic/externalFeed/ExternalFeedBlockPlugin.inc.php

```diff
@@ -89,7 +96,7 @@ function getContents(&$templateMgr, $request = null) {
 
 		$requestedPage = $request->getRequestedPage();
 		$externalFeedDao = DAORegistry::getDAO('ExternalFeedDAO');
-		$plugin->import('simplepie.SimplePie');
+		require_once('lib/pkp/lib/vendor/simplepie/simplepie/autoloader.php');
```

Comment:
Now that we've got autoloading in place, I don't think this is necessary any more.